specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2020-09-01/cosmos-db.json

```diff
@@ -6327,6 +6337,11 @@
           "description": "Flag to indicate whether to enable storage analytics.",
```

Comment:
I wonder if we can club enableAnalyticalStorage and the analyticalStorageConfig in one object. Let us discuss the possibilities when we review this in RP review meeting